Sample Data/Custom/Auth0AM_CL.json

```diff
@@ -0,0 +1,121 @@
+      "log_id":"LOG_ID",
+      "_id":"_ID",
+      "isMobile":false
+   },
+   {
+      "date":"2022-04-13T11:33:18.757Z",
+      "type": "sapi",
+      "description": "Create client grant",
+      "client_id": "CLIENT_ID",
+      "ip": "35.160.3.103",
+      "user_agent": "Chrome 100.0.4896 / Mac OS X 10.15.7",
+      "user_id": "USER_ID",
+      "log_id": "LOG_ID",
+      "_id": "_ID",
+      "isMobile_b": "false",
+      "details_request_method": "post",
+      "details_request_path": "/api/v2/client-grants",
+      "details_request_userAgent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/100.0.4896.75 Safari/537.36",
+      "details_request_channel": "https://manage.auth0.com/",
+      "details_request_ip": "35.160.3.103",
+      "details_request_auth_user_user_id": "AUTH_USER_ID",
+      "details_request_auth_user_name": "USER NAME",
+      "details_request_auth_user_email": "EMAIL",
+      "details_request_auth_strategy": "jwt",
+      "details_request_auth_credentials_jti": "CLIENT_ID",
+      "details_response_statusCode": "201",
```

Comment:
@ospeka, as the value of "details_response_statusCode" is given with "" it is saved as string, please check and update, thanks.